core/lib/Plugins.ts

```diff
@@ -164,11 +190,26 @@ export default class Plugins implements IPlugins {
   // ADDING PLUGINS TO THE STACK
 
   public use(Plugin: IPluginClass) {
+    if (this.instanceById[Plugin.id]) return;
     this.addPluginInstance(new Plugin(this.createOptions));
   }
 
   private addPluginInstance(plugin: IPlugin) {
     this.instances.push(plugin);
     this.instanceById[plugin.id] = plugin;
   }
+
+  private require(pluginId: string): IPluginClass {
+    if (!Plugins.pluginClassesById[pluginId]) {
+      try {
+        // eslint-disable-next-line global-require,import/no-dynamic-require
+        const Plugin = require(pluginId);
```

Comment:
Does this need to have the .default check to handle typescript generated default exports?

Reply:
Yes. Will push a new commit.

Reply:
Pushed.